website/docs/docs/core/connect-data-platform/profiles.yml.md

```diff
@@ -28,6 +28,7 @@ This section identifies the parts of your `profiles.yml` that aren't specific to
   [debug](/reference/global-configs/logs#log-level): <true | false>
   [version_check](/reference/global-configs/version-compatibility): <true | false>
   [fail_fast](/reference/global-configs/failing-fast): <true | false>
+  [indirect_selection](/reference/global-configs/indirect-selection): <eager | cautious | buildable | empty>
```

Comment:
We might want to move all of these flags into the project file

Reply:
Good call!

It's covered in #4814 ([preview](https://docs-getdbt-com-git-jerco-flags-behavior-changes-dbt-labs.vercel.app/reference/dbt_project.yml)) with a link to `<global-configs>` like this:

<img width="578" alt="image" src="https://github.com/dbt-labs/docs.getdbt.com/assets/44704949/9b78de52-cac5-4fcb-93ac-f001c9bc1a41">